### DISCUSSION TITLE
Windows Terminal v1.19.10573.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#16776] [Announcements]
This is a quick servicing release to cross a few eyes and dot a few Ts.

## Changes

* Dashed, dotted and curly underlines have been changed to be slightly more readable (and pretty!) (#16719)

## Bug Fixes

* In CMD, pressing <kbd>Tab</kbd> in the middle of a line will _finally_ clear the remainder of the line from the screen (#16718)
* We have removed one source of graphical issues on x86, which was caused by a compiler bug (#16742)
* You can once again copy/paste entire files onto the Terminal window (#16634)

---

_Binary files inside the unpackaged distribution archive bear the version number `1.19.240226003`._

### Asset Hashes

- Microsoft.WindowsTerminal_1.19.10573.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `A89F03E264813E454B3E1674886843294A4F5C46DF0A6F5984AC7FB8E709D1E2`
- Microsoft.WindowsTerminal_1.19.10573.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `59396AF53C987B3385FDB26A97CA026C8F1FA892FB23E9F3AA7DEC53E7F7DB0A`
- Microsoft.WindowsTerminal_1.19.10573.0_arm64.zip
   - SHA256 `A0E71AF2D0E0BDDE82F861CE7A9061D35B82C4A2FFCE76DEC06CAB4234E772CE`
- Microsoft.WindowsTerminal_1.19.10573.0_x64.zip
   - SHA256 `F756A41FA2DBEE274334CB49D93A84CB29E5DF0A2446FC79BF7ED9FFE8B49FFB`
- Microsoft.WindowsTerminal_1.19.10573.0_x86.zip
   - SHA256 `9920A17DB43D58E2561F7271AC3AD9BB0C05E97CA946BE50422301227153E5EA`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.19.10573.0'>Windows Terminal v1.19.10573.0</a>.</em>

**Comment by velomatt** [comment]
Anyone having trouble with Quake Mode with this release? `WIN-~` is just spawning multiple new instances of it, instead of showing/hiding the single instance.

`ALT-space-N` will hide the Quake Mode window, but you cannot easily recall it. It is also visible in the task bar after doing so.

**Comment by ZhangTianrong** [comment]
Windows terminal crashes when opening a new tab no matter which profile (including the same profile as the first tab) it is running since this version for me. Running in a separate window is fine.

**Reply by DHowett** [reply]
Upgrade MacType 🙂

**Comment by Idighekere** [comment]
i get this error when i try to open Windows Terminal after i downloaded

the procedure entry point iswow64process2 could not be located in the dynamic link library C:\Users\Murphy\Downloads\Microsoft.WindowsTerminal_1.19.10573.0_x86\terminal-1.19.10573.0\WindowsTerminal.exe

**Reply by DHowett** [reply]
What version of Windows are you using?

**Reply by Idighekere** [reply]
10.0.10240 Build 10240

**Reply by DHowett** [reply]
That is the oldest existent version of Windows 10. It hasn’t been meaningfully updated since 2016. Terminal requires version 10.0.19041 or greater!
